### DISCUSSION TITLE
Welcome to transmission Discussions!

### DISCUSSION THREAD

**Discussion by ckerr** [transmission/transmission#1897] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GaryElshaw** [comment]
Oh! I Just said this.

I forgot to ask, how can I help? I have some time and used to be one of the Mac Firefox testers 1.6 - 2.0 before the versioning went crazy. Although I don't code, I have experience developing UX/UI for tablet applications. This is some of my volunteer work I did in Cambodia. https://www.youtube.com/watch?v=kcwfqNvllyU

**Comment by joaotorres** [comment]
Hello! I'm Joao, longtime Transmission user and also a Software Engineer. Had an idea for a behavior change suggestion for the app and thought I'd look up the best place to give it. That's how I ended up here :)

**Comment by mikexryan** [comment]
I want to ask a question: is there a setting in Transmission that will halt, pause or stop transmission activity if you ARE NOT connected to a VPN? If so, where is it? Example, you are downloading using a VPN and then, by accident, you shut off the VPN and forget to turn the VPN back on? I have only done this once but have received a nasty message from internet provider and want to prevent that. Note there are times when I have to shut off the VPN to access a site that won't let me connect when a VPN is in use. I then forget that the VPN is not on but Transmission continues to download data that can be traced/tracked by my internet provider.

**Reply by valcomm** [reply]
> I want to ask a question: is there a setting in Transmission that will halt, pause or stop transmission activity if you ARE NOT connected to a VPN?

That is not a Transmission issue, but a VPN user's.
VPN on/off detection could be done in many ways.  And actions to be do done differ a lot.  Covering those would put a configuration complexity on user comparable to writing a side script that does required detection and actions.
My version of the script is 140 lines long, does pretty interesting things that I need and saved me many times from the failed VPN due to different reasons (last one was caused by provider changes when they interrupted service in the middle of the night).
